modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -0,0 +1,260 @@
+import Panel from 'Panel';
+/**
+ * New Profile Form
+ *
+ * Create a new profile form
+ *
+ * @author  Shen Wang
+ * @version 1.0.0
+ * */
+class NewProfileIndex extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: {},
+            configData: {},
+            formData: {},
+            newData: {},
+            isLoaded: false,
+            isCreated: false,
+            errMessage: '',
+        };
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.setFormData = this.setFormData.bind(this);
+        this.fetchData = this.fetchData.bind(this);
+    }
+
+  componentDidMount() {
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}));
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   * Additionally add hiddenHeaders to global loris variable
+   * for easy access by columnFormatter.
+   *
+   * @return {object}
+   */
+  fetchData() {
+    return fetch(this.props.dataURL, {credentials: 'same-origin'})
+      .then((resp) => resp.json())
+      .then((data) => this.setState({configData: data}))
+      .catch((error) => {
+        this.setState({error: true});
+        console.error(error);
+      });
+  }
+
+    validate() {
+        const data = this.state.formData;
+        let isError = false;
+        if (data.dateTaken !== data.dateTakenConfirm) {
+            isError = true;
+        }
+        let decError = false;
+        if (this.state.configData['edc'] === 'true'
+            && data.edcDateTaken !== data.edcDateTakenConfirm
+        ) {
+            decError = true;
+        }
+        return isError || decError;
+    }
+
+
+    /**
+   * Handles form submission
+   *
+   * @param {event} e - Form submition event
+   */
+    handleSubmit(e) {
+        e.preventDefault();
+        const err = this.validate();
+        this.setState(
+            {
+                errMessage: '',
+                }
+        );
+        if (err) {
+            this.setState(
+                {
+                    errMessage: 'Date of birth fields must match',
+                    isCreated: false,
+                }
+            );
+        } else {
+    let formData = this.state.formData;
+    let formObject = new FormData();
+    for (let key in formData) {
+      if (formData[key] !== '') {
+        formObject.append(key, formData[key]);
+      }
+    }
+/* eslint-disable no-console */
```

Comment:
Remove this (it also doesn't do anything since there's not any console.log in this section anyways..)